ctapipe/instrument/guess.py

```diff
@@ -0,0 +1,55 @@
+from collections import namedtuple
+import astropy.units as u
+
+
+GuessingKey = namedtuple('GuessingKey', ['n_pixels', 'focal_length'])
+GuessingResult = namedtuple(
+    'GuessingResult', ['type', 'name', 'camera_name', 'n_mirrors']
+)
+
+
+TELESCOPE_NAMES = {
+    GuessingKey(2048, 2.28): GuessingResult('SST', 'GCT', 'CHEC', 2),
+    GuessingKey(2368, 2.15): GuessingResult('SST', 'ASTRI', 'ASTRICam', 2),
```

Comment:
Can an entry be added for:
`GuessingKey(2048, 2.15): GuessingResult('SST', 'ASTRI', 'CHEC', 2),`

Reply:
done